src/machine/behavior/is/mod.rs

```diff
@@ -288,3 +291,262 @@ pub const SEMANTICS: [Semantic; 256] = [
     other::get,
     interrupt::trip,
 ];
+
+const INSTR_NAMES: [&'static str; 256] = [
+    "interrupt::trap",
```

Comment:
It was just faster to just copy the SEMANTICS array and put quotation marks around the names.